internal/fetchers/oci/client_test.go

```diff
@@ -0,0 +1,88 @@
+// Copyright 2023 Red Hat, Inc.
```

Comment:
`Copyright The Enterprise Contract Contributors`

We should probably fix it on the other files under fetchers as well. I think those were missed after the mass update.